src/gui/wizard/flow2authwidget.ui

```diff
@@ -115,15 +161,56 @@
         </spacer>
        </item>
        <item>
-        <layout class="QVBoxLayout" name="verticalLayout_2">
-         <property name="topMargin">
-          <number>0</number>
-         </property>
+        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
-          <widget class="OCC::LinkLabel" name="copyLinkLabel" native="true"/>
+          <widget class="QPushButton" name="copyLinkLabel">
+           <property name="sizePolicy">
+            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
+             <horstretch>0</horstretch>
+             <verstretch>0</verstretch>
+            </sizepolicy>
+           </property>
+           <property name="font">
+            <font>
+             <family>Segoe UI</family>
+             <pointsize>9</pointsize>
```

Comment:
custom font again (not sure if we should also keep the smaller font size for the buttons)

```suggestion
             <pointsize>9</pointsize>
```